### DISCUSSION TITLE
Feature Request: labeling each output, and tabs for different kinds of outputs

### DISCUSSION THREAD

**Discussion by sarmenb** [spatie/ray#410] [Ideas]
here are a few features i would like to see on ray. not sure if im already capable of doing it but let me know.  I will be refering to laravel but im sure it can be helpful in other projects.

1) ability to label the output displaying in ray. for example:

ray('cart data', $somearray); -> so that i know which array it is showing.

2) tabs for different outputs. for example if i wanted to see events in one tab and queries in another etc.

**Comment by anugupta** [comment]
Yes, it would very useful to be able to add a label to the timestamp, like the `->table()` label feature does. Something like `ray($collection)->label('Beatles Records')` would be nice, I think.

**Reply by freekmurze** [reply]
We'll add this later this week 👍

**Reply by anugupta** [reply]
Amazing, thanks 🚀🎉
